crates/bevy_asset/src/processor/mod.rs

```diff
@@ -1,3 +1,25 @@
+//! Asset processing in Bevy is a framework by which artist-authored assets can be automatically prepared for use in game.
+//!
+//! You can think of the asset processing system as a "build system" for assets.
+//! When an artist adds a new asset to the project, the asset processing system will automatically perform the specified processing steps on the asset.
+//! This can include things like converting a `.psd` file to a `.png`, compressing a `.wav` file to an `.ogg`, or generating mipmaps for a texture.
+//!
+//! Its core values are:
+//!
+//! 1. Automatic: changing the artist-format asset should not require any manual steps to get it back in the game.
+//! 2. Configurable: every game has its own needs, and a high level of transparency and control is required.
+//! 3. Lossless: the original asset should always be preserved, avoiding frustration and making experimentation safe.
```

Comment:
```suggestion
//! 3. Lossless: the original asset should always be preserved,... avoiding frustration and making experimentation safe.
```

Don't like the part after ..., doesn't really make sense to me. You don't want to modify the existing asset so that users can change it later, idk a good way to phrase it.

Reply:
Yeah, I think we should try and phrase this more clearly. You a) want to make sure that changes to asset processing can be tested safely b) not muck with the original asset so artist can go back to modifying it freely.